src/extension.ts

```diff
@@ -24,6 +24,15 @@ export async function activate(context: ExtensionContext): Promise<void> {
     // Mark the app as already initialized
     store.manifestSelected(manifest)
 
+    // Watch for workspace config changes
+    workspace.onDidChangeConfiguration(event => {
+      // Reinitialize when settings have changed
+      const affected = event.affectsConfiguration(`${EXT_ID}`, (workspace.workspaceFolders as WorkspaceFolder[])[0].uri);
+      if (affected) {
+        store.initialize()
+      }
```

Comment:
Not sure about this part, do we always need to initialize even if the user disables the settings?